webplatform/Initial application page html/learn.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title> Intitial page application-period</title>
+<style type="text/css">
+
+.Header img {
```

Comment:
let's not use capital letters when naming css classes or ids :) Also because we only have one header I think you can use id here